src/components/Metrics.js

```diff
@@ -0,0 +1,120 @@
+import { useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { useSelector, useDispatch } from 'react-redux';
+import { ArrowBackIos, Settings, Mic } from '@material-ui/icons';
+import ArrowForwardIcon from '@material-ui/icons/ArrowForward';
+import './Metrics.css';
+import homeImg from '../assets/images/europe.png';
+import { fetchCountries } from '../redux/metrics/Metrics';
+
+export const formatNumber = (num) => num.toString().replace(/(\d)(?=(\d{3})+(?!\d))/g, '1,');
+
+const corona = {
+  backgroundImage: `url(${homeImg})`,
+  backgroundPosition: 'center',
+  backgroundSize: 'cover',
+  backgroundRepeat: 'no-repeat',
+};
+
+const Item = ({ confirmed, name }) => (
+  <div className="country-ite">
+    <div className="arrow-forward-btn">
+      <ArrowForwardIcon />
+    </div>
+    <div className="corona"> </div>
+    <div className="country-title">
+      <div className="confirmed-cases">
+        <h4 className="">{name}</h4>
+        <p className="para">{formatNumber(confirmed)}</p>
+      </div>
+    </div>
+  </div>
+);
+
+const Grid = ({ items = [] }) => (
+  <div className="country-cont">
+    <h4 className="stats">STATS BY COUNTRY</h4>
+    <div className="country-data">
+      {items.map(({ name, confirmed }) => (
+        <div key={name} className="even">
+          <div style={corona} className="country">
+            <Link to={`/country/${name}`}>
+              <Item confirmed={formatNumber(confirmed)} name={name} />
+            </Link>
+          </div>
+        </div>
+      ))}
+    </div>
+  </div>
+);
+
+const Metrics = () => {
```

Comment:
- [ ] Your project looks very nice! As per the project requirement, you should add a Filter stateless component to filter data retrieved from the API. For example, you can add a functionality to filter the data by country, so a user can search for a country

Reply:
Hello @Georjane, thanks for the review, please try to explain to me what you mean. Are you saying that you did not navigate to the details page or what? The data is being filtered that's the reason why I created two fetch functions to filter by continent or by country

Reply:
Hi, @Georjane, data is made available through props